crates/libcontainer/src/syscall/linux.rs

```diff
@@ -19,14 +20,41 @@ use nix::{
     unistd,
     unistd::{chown, fchdir, pivot_root, setgroups, sethostname, Gid, Uid},
 };
-use syscalls::{syscall, Sysno::close_range};
+use syscalls::{syscall, Sysno, Sysno::close_range};
 
 use oci_spec::runtime::LinuxRlimit;
 
 use super::Syscall;
 use crate::syscall::syscall::CloseRange;
 use crate::{capabilities, utils};
 
+// Constants used by mount_setattr(2).
+pub const MOUNT_ATTR_RDONLY: u64 = 0x00000001; // Mount read-only.
+pub const MOUNT_ATTR_NOSUID: u64 = 0x00000002; // Ignore suid and sgid bits.
+pub const MOUNT_ATTR_NODEV: u64 = 0x00000004; // Disallow access to device special files.
+pub const MOUNT_ATTR_NOEXEC: u64 = 0x00000008; // Disallow program execution.
+pub const MOUNT_ATTR__ATIME: u64 = 0x00000070; // Setting on how atime should be updated.
+pub const MOUNT_ATTR_RELATIME: u64 = 0x00000000; // - Update atime relative to mtime/ctime.
+pub const MOUNT_ATTR_NOATIME: u64 = 0x00000010; // - Do not update access times.
+pub const MOUNT_ATTR_STRICTATIME: u64 = 0x00000020; // - Always perform atime updates.
+pub const MOUNT_ATTR_NODIRATIME: u64 = 0x00000080; // Do not update directory access times.
+pub const MOUNT_ATTR_NOSYMFOLLOW: u64 = 0x00200000; // Prevents following symbolic links.
+pub const AT_RECURSIVE: u32 = 0x00008000; // Change the mount properties of the entire mount tree.
```

Comment:
Why don't you implement it as enum and implement `From<str>` trait?

Reply:
About enum member name, which one is better snake case or camel case.

- snake case
  ```
  #[repr(u64)]
  pub enum MountAttr {
      #[allow(non_camel_case_types)]
      MOUNT_ATTR_RDONLY = 0x00000001,
  -- snip --
  ```
- camel case
  ```
  #[repr(u64)]
  pub enum MountAttr {
      MountAttrRdonly = 0x00000001,
  -- snip --
  ```

Reply:
I prefer camelCase.

Reply:
fixed